### PR TITLE
[node-headless-ssr-exp-edge][Angular] Danish language is not rendered when refresh the page

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Description / Motivation
* Take `language` from layout data, instead of `viewBag`, it doesn't contain `language`

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/angular/server.bundle.ts [hunk 1]
```diff
@@ -52,13 +52,17 @@ function renderView(
...
     if (parsedData) {
       state.sitecore = parsedData.sitecore;
+      state.language = parsedData.sitecore.context.language;
     }
 
     // parse the URL that's being handled by Sitecore so we can pass in the initial state to the app
     const routeParser = new JssRouteBuilderService();
     const jssRoute = routeParser.parseRouteUrl(path.split('/').filter((segment) => segment));
     state.serverRoute = jssRoute.serverRoute;
-    state.language = parsedViewBag.language || jssRoute.language;
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/templates/angular/server.bundle.ts:61]: In the case of Experience Editor / Horizon, the viewbag would actually have the language.
https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/testData/ee-data.ts#L12

Comment (ambrauer) [inline packages/create-sitecore-jss/src/templates/angular/server.bundle.ts:61]: In the case of Experience Editor / Horizon, the viewbag would actually have the language.
https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/testData/ee-data.ts#L12

Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/templates/angular/server.bundle.ts:61]: @ambrauer Usually we don't take language from the viewBag, and seems like it's not provided in other cases. Anyway we should have language in the LayoutData, right?

Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/templates/angular/server.bundle.ts:61]: Usually we take only dictionary, `node-headless-ssr-proxy` also returns `viewBag` without the `language`

Comment (ambrauer) [reply packages/create-sitecore-jss/src/templates/angular/server.bundle.ts:61]: @illiakovalenko You're probably right. I remember using it in the [nextjs editing middleware](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/middleware/editing-render-middleware.ts#L240), but we could probably use the LayoutData value here as well. React and Vue I assume are working fine in editing mode without using viewbag language.

**Review by ambrauer — APPROVED** [review]

**Pull request merged**